src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -87,6 +87,22 @@ export const QueryResultPane = () => {
 			renderHeaderCell: () => <>{MESSAGE}</>
 		}),
 	];
+	const gridParentRef = useRef<HTMLDivElement>(null);
+	const ribbonRef = useRef<HTMLDivElement>(null);
+	// Resize grid when parent element resizes
+	useEffect(() => {
+		const gridParent = gridParentRef.current;
+		if (!gridParent) {return;}
+        const observer = new ResizeObserver(() => {
```

Comment:
Please make sure we are disposing these properly

Reply:
I disconnect the observer afterwards, is there something additional needed?

Reply:
That should be enough imo